### PR TITLE
Fix custom strategy bug + general clean up

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

- fix bug where selecting `custom strategy` breaks the page
- clean up ballot strategy utils/constants
- use aliased components/ directory imports for shared components (where it makes sense)

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../CONTRIBUTING.md#browser-support).

### CODE REVIEW

File: src/components/shared/forms/RulesForm.tsx [hunk 1]
```diff
@@ -89,7 +92,7 @@ export default function RulesForm({
           </Trans>
         </p>
 
-        {ballotStrategies()[selectedIndex ?? 0].address ===
+        {BALLOT_STRATEGIES[selectedIndex ?? 0]?.address ===
```
Comment (aeolianeth) [inline src/components/shared/forms/RulesForm.tsx:95]: This is the bug fix

**Comment by aeolianeth** [comment]
@peripheralist @johnnyd-eth could ya'll pls review?

**Comment by peripheralist** [comment]
hm, i'm confused why the bug was happening before.

iirc @johnnyd-eth organized those files/strategies the way they were to avoid some bugs with lingui not working with them, although i don't think we ever figured out what the real issue was.

**Comment by aeolianeth** [comment]
@peripheralist good call-out, in any case, I think @johnnyd-eth is fixing this in https://github.com/jbx-protocol/juice-interface/pull/608, so I'll close this for now

**Pull request closed without merging**